### ISSUE TITLE
ASUS G15 (GA503Q) Support

### ISSUE THREAD

**Issue by hughesjs** [pop-os/pop#1555]
Trying to get Pop OS installed on the new ASUS G15.

The GUI fails to start on boot, I can get into a different Terminal using Ctrl+Alt+Num however.

It seems as though the issue is XOrg related as running xrandr -q fails to find any displays.

Let me know what you need off me to debug this and I'll update this.

**Comment by bflanagin** [comment]
@hughesjs: Which version of Pop!_OS did you try? Was it the Intel version of Pop or Nvidia?

**Comment by hughesjs** [comment]
Pretty sure I tried both last night, but I'll give it another go to check I didn't accidentally flash the same one twice

**Comment by bflanagin** [comment]
@hughesjs that's what I'm guessing as well. There have been some updates to the nvidia driver since we released the iso's and that specific card may have had issues with the driver provided. If you can get the system running with the Intel iso you can install the newest Nvidia drivers after Pop!_OS is installed with the following command.

`sudo apt install system76-driver-nvidia`

**Comment by hughesjs** [comment]
So I've just given that a go and the graphical environment fails to start after installing that package and rebooting. There's no error shown, it just sticks on a blinking cursor on a black screen. 

I can, however, still ctrl+alt+num into another vterm.

**Comment by bflanagin** [comment]
Tough break! We're getting ready to release the newest Nvidia driver in the next few days. Hopefully that will solve the issue. Until then you should be able to revert to the open source driver by running 

'sudo apt purge nvidia-*` Your computer wont be at 100% but at least it will be usable.  I'll try to update this issue when the new driver lands, assuming you don't get to it first.

**Comment by hughesjs** [comment]
Is there a beta build I can try? Happy to help you get this tested.

**Comment by bflanagin** [comment]
Not really, but you can manually install the diver by downloading the installer:
https://us.download.nvidia.com/XFree86/Linux-x86_64/460.56/NVIDIA-Linux-x86_64-460.56.run

Its not a horrible process, but its not as streamlined as the one we offer, and I can't promise it won't cause issues at some point down the road. (I've never had an issue with it, but there is always a chance.) It includes some fixes to the 3000 series which might get yours going. You'll want to uninstall the current drivers as I suggested above to be sure its cleanly installed.

Not knowing how comfortable you are on the command line here is how

**Comment by hughesjs** [comment]
I'll give it a shot and see if it's any better.

I'm a Software Engineer who usually uses Arch, so no worries on the terminal front :)

**Comment by bflanagin** [comment]
Great! Also the Nvidia update was merged yesterday evening, so you should see it in your standard updates soon.

**Comment by DosPuppet** [comment]
Same here, but there is another issue with the igpu, no drivers in kernel for the igpu with ryzen 5900x. Should be ok with kernel 5.11, but no boot at after updating kernel to 5.11, had to reinstall pop os.

**Comment by DosPuppet** [comment]
OK, got it fixed. The issue is not due to Nvidia drivers but to messed up AMD cezanne drivers. Used this to fix (https://forums.developer.nvidia.com/t/ubuntu-mate-20-04-with-rtx-3070-on-ryzen-5900-black-screen-after-boot/167681), everything is back to normal, pop OS up and running on GA503Q. Need to install some asus fixes for keyboard and sound.

**Comment by santi921** [comment]
@DosPuppet did you have to upgrade the kernel from the one in 20.10 to get this solution to work? I tried moving to kernel 5.10 but everything broke

**Comment by foundObjects** [comment]
Just a heads up to anyone struggling with booting Cezanne: you probably need more recent amd microcode. I can't give a detailed fix since I'm running Arch on mine but I had to build my linux firmware and amd ucode packages from git and then update all of my kernel ramdisks to even be able to boot my Cezanne machine using 5.10 or 5.11.

**Comment by DosPuppet** [comment]
@santi921  yes I upgraded, must more importantly, like @foundObjects said, i upgraded amd firmware and updated ramdisks for the upgrade before rebooting. Everything runs smooth after that (every step is in the post link i mentionned).

**Comment by saadrupai97** [comment]
I have installed PoP(Nvidia) but the keyboard light is not being lit up. My laptop model is asus rog strix G153QE, In addition,  the various modes I was being able to use on windows are not found on pop

**Comment by hughesjs** [comment]
For anyone still having issues, can I point you towards [this site](https://asus-linux.org/) they maintain a custom kernel with the extra modules for this laptop and have some handy utilities for managing the machine.
